### DISCUSSION TITLE
"Used by counter" not showing all projects (and main one)

### DISCUSSION THREAD

**Discussion by janpio** [community/community#21786] [New to GitHub]
<p><a href="https://github.community/t/used-by-counter-not-showing-main-project/2654">Similar to this older topic</a>, our projects is unfortunatly not showing all npm packages in the dropdown to configure the “Used by counter”:</p>
<p><a href="https://github.com/prisma/prisma2" rel="nofollow noopener">https://github.com/prisma/prisma2</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/6ee6c1b6f84b3a1204b69eefc7dc32575bdbd3ae.png" data-download-href="/uploads/short-url/fP4ULVLSnV9Ixpd1zpnuCXSD8OG.png?dl=1" title="asd.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/183673/181066880-a35faea4-225b-43a6-afb6-197d4b6c7039.png" alt="asd.PNG" data-base62-sha1="fP4ULVLSnV9Ixpd1zpnuCXSD8OG" width="439" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/6ee6c1b6f84b3a1204b69eefc7dc32575bdbd3ae_2_439x500.png, https://aws1.discourse-cdn.com/github/original/1X/6ee6c1b6f84b3a1204b69eefc7dc32575bdbd3ae.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/6ee6c1b6f84b3a1204b69eefc7dc32575bdbd3ae.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/6ee6c1b6f84b3a1204b69eefc7dc32575bdbd3ae_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">asd.PNG</span><span class="informations">516×587 35.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The package we are actually interested in showing is defined here: <a href="https://github.com/prisma/prisma2/blob/master/cli/prisma2/package.json" rel="nofollow noopener">https://github.com/prisma/prisma2/blob/master/cli/prisma2/package.json</a></p>
<p>What can we do?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/janpio">@janpio</a>,</p>
<p>Thank you for being here! I’ve associated the package for you, would you mind trying to assign it again and let me know how you get on?</p>

**Comment by janpio** [comment]
<p>Thanks <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> - that worked like a charm.</p>
<p>(We’ll rename both the repo and the package in a few days, so that will be interesting. But I know how to reach out now if it does not go well.)</p>
